pkg/mons/restore_quorum.go

```diff
@@ -278,13 +279,15 @@ func getMonDetails(goodMon string, monEndpoints []string) ([]string, string, str
 	return badMons, goodMonPublicIp, goodMonPort, nil
 }
 
-func promptToContinueOrCancel(answer string) (string, error) {
+func PromptToContinueOrCancel(inputSource, expectedAnswer, answer string) (string, error) {
```

Comment:
Just return an error. The caller can choose to log something specific, no need to return it here.
```suggestion
func PromptToContinueOrCancel(inputSource, expectedAnswer, answer string) error {
```